sycl/include/CL/sycl/INTEL/ac_int.hpp

```diff
@@ -0,0 +1,2448 @@
+//==----------------- ac_int.hpp --- SYCL FPGA AC data-type ----------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// Algorithmic C (tm) Datatypes
+//
+// Software Version: 3.7
+//
+// Release Date    : Wed Jun  1 13:21:52 PDT 2016
+// Release Type    : Production Release
+// Release Build   : 3.7.0
+//
+// Copyright 2004-2016, Mentor Graphics Corporation,
```

Comment:
This file has this Mentor Graphics Copyright message, then the Apache License at the first 5 lines.
I don't know how these 2 work together.
@bader - please advise on how to proceed or recommend someone who is expert is such questions.

Reply:
It would be nice if as a side effect of this project we can get a common open-source modern C++ header across various HLS implementations, including improving back https://github.com/hlslibs/ac_types
But by browsing https://github.com/hlslibs/ac_types/blob/master/pdfdocs I see "No C++11 features or dependencies." Ouch! They might have some exciting life there and are probably not interested by any improvement...

Reply:
> This file has this Mentor Graphics Copyright message, then the Apache License at the first 5 lines.
> I don't know how these 2 work together.
> @bader - please advise on how to proceed or recommend someone who is expert is such questions.

@pvchupin, please, help with clarifying this comment.

Reply:
Can anyone explain the Protex IP Scan build test failure?
Initially I suspected the dual license was causing it however:
 1. Log states "There are no license conflicts in this project."
 2. The test fails even if I update the file to only keep the Apache License and remove all the Mentor Graphics License text at the start (tested on an internal repository).